### PR TITLE
Add success

### PR DESCRIPTION
**Pull request by JennaDalgety**
- cleaned up vue editor toolbar on faq and success components
- removed extraneous code from faq components
- added working success (add/edit/delete) components and functionality
- will add image uploading functionality to success edits/adds next

### CODE REVIEW

File: src/components/successEdit.vue [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+      <router-link :to="'successAddNew'" class="btn btn-success">Add New Success Story</router-link>
+    </div>
+  </div>
+  <div class="row">
+    <div class="row mx-0 my-3" v-for="(successStory, index) in successStories" :key="index">
+      <form>
+        <div class="form-group row">
+          <label class="col-md-2 col-form-label text-right">About Camper</label>
+          <div class="col-md-10">
+            <input readonly type="text" class="form-control" v-model="successStory.about" v-bind:placeholder="successStory.about">
```
Comment (knightofknee) [inline src/components/successEdit.vue]: this should be a div like the below one. also consider having the max height be bigger, as longer stuff that gets cut off isn't terribly readable, and I don't see a big downside to having it bigger

**Review by knightofknee — APPROVED** [review]
looks good except that one text box not like the other, with my comment

File: src/components/successAddNew.vue [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+            <vue-editor id="newStoryAbout" v-model="newStory.about" :editorToolbar="customToolbar"></vue-editor>
+          </div>
+        </div>
+        <div class="form-group row">
+          <label class="col-md-2 col-form-label text-right">What They Learned</label>
+          <div class="col-md-10">
+            <vue-editor id="newStoryLearned" v-model="newStory.learned" :editorToolbar="customToolbar"></vue-editor>
+          </div>
+        </div>
+        <!-- <div class="form-group row">
```
Comment (jdaniel0x50) [inline src/components/successAddNew.vue]: This <div> is commented out - should you just remove it?

File: src/components/successAddNew.vue [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+  data() {
+    return {
+      newStory: {},
+      messages: false,
+      errors: [],
+      customToolbar: [
+        ['bold', 'italic', 'underline'],
+        [{ list: 'ordered' }, { list: 'bullet' }]
+      ]
+      // cloudinary: {
```
Comment (jdaniel0x50) [inline src/components/successAddNew.vue]: commented out - remove?

File: src/components/successAddNew.vue [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+      //   uploadPreset: 'loazbic8',
+      //   apiKey: '234871425639756',
```
Comment (jdaniel0x50) [inline src/components/successAddNew.vue]: definitely remove the api key -- this should not be on github!

File: src/components/successAddNew.vue [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+      //   cloudName: 'wernextgeneration'
+      // },
+      // file: [],
+      // imageData: ''
+    };
+  },
+  methods: {
+    // previewImage: function(event) {
```
Comment (jdaniel0x50) [inline src/components/successAddNew.vue]: also commented out?

File: src/components/successAddNew.vue [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+        return true;
+      }
+      if (!this.newStory.about)
+        this.errors.push('About Camper Section Required');
+      if (!this.newStory.learned)
+        this.errors.push('What They Learned Section Required');
+    }
+  }
+};
+// submitImage: function(evt) {
```
Comment (jdaniel0x50) [inline src/components/successAddNew.vue]: also commented out

**Review by jdaniel0x50 — CHANGES_REQUESTED** [review]
Looks good. I made some comments where there remain significant portions of code commented out - that should likely be removed if they are no longer needed. The most significant comment is to remove the api key from the code base - and pass it through the database and data layer so that it is not searchable on our open source github account

**Pull request merged**